ui/src/pages/Dashboard/dashboardStyle.css

```diff
@@ -328,3 +328,95 @@
     grid-row: 3 / 6;
     margin-top: 16px;
 }
+
+/* resource tab */
+
+#resource_tab_label_viz {
+    min-width: 450px;
+}
+
+[data-input-id='resource_tab_input'] {
+    grid-column: 6;
+    align-content: flex-end;
+    grid-row: 1;
+    place-self: end;
+    margin-top: 32px;
+
+    [data-test='input-container'] {
+        display: inline-flex;
+        padding: 0;
+    }
+    [data-test='input-title-remove-container'] {
+        margin: 5px;
+    }
+    [data-test='input-title'] {
+        font-size: 14px;
+    }
+}
+
+#resource_tab_description_viz {
+    grid-column: 1 / span 4;
+    grid-row: 1;
+    height: fit-content;
+    margin-top: 32px;
+}
+
+#resource_tab_timerange_label_start_viz {
+    grid-column: 1;
+    grid-row: 1;
+    inline-size: fit-content;
+    width: 145px;
+    margin-top: 35px;
+
+    [data-test='majorValue'] {
+        font-size: 14px;
+        line-height: 14px;
+        font-family: 'Splunk Platform Sans', 'Splunk Platform Sans', 'Proxima Nova', Roboto, Droid,
+            'Helvetica Neue', Helvetica, Arial, sans-serif;
+        font-weight: 400;
+        color: rgb(108, 118, 133);
```

Comment:
we will have a hard time with dark and light mode

Reply:
11 copy pastas?
<img width="600" alt="image" src="https://github.com/splunk/addonfactory-ucc-generator/assets/142901247/2eb54b7e-2795-48f4-a068-68d4e2bef1be">

Reply:
similar problem as with background added in style.css in main file, maybe in next sprint if some time pops up i will have a look at those themes in ucc